netlify.toml

```diff
@@ -620,4 +620,48 @@
 # Added: 2021-07-28
 [[redirects]]
     from = "/docs/self-host/deploy/qovery"
-    to = "/docs/self-host/deploy/other"
\ No newline at end of file
+    to = "/docs/self-host/deploy/other"
+# Added: 2021-08-02
+[[redirects]]
```

Comment:
Hmmm did you need to make changes to these redirects? Or is this something funky Git thinks you changed?

Reply:
This is fixed now 😄